src/sst/elements/vanadis/Makefile.am

```diff
@@ -602,13 +601,15 @@ libvanadis_la_SOURCES = \
 	vanadis.cc \
 	$(VANADIS_SRC_FILES)
 
-#libvanadisdbg_la_SOURCES = \
-#	vanadisdbg.cc \
-#	$(VANADIS_SRC_FILES)
+libvanadisdbg_la_SOURCES = \
+	vanadis.cc \
+	$(VANADIS_SRC_FILES)
 
 libvanadis_la_LDFLAGS = -module -avoid-version
 
-#libvanadisdbg_la_LDFLAGS = -module -avoid-version
+libvanadisdbg_la_LDFLAGS = -module -avoid-version
+
+libvanadisdbg_la_CPPFLAGS = -DVANADIS_BUILD_DEBUG $(AM_CPPFLAGS)
```

Comment:
Are these flags correct? I think you need to replicate them for both libvanadis_la and libvanadisdbg_la

Reply:
@gvoskuilen I think you're right and aree ok with the implicit AM_CPPFLAGS for libvanadis.